src/ukify/ukify.py

```diff
@@ -706,6 +742,56 @@ def make_uki(opts):
     print(f"Wrote {'signed' if sign_args_present else 'unsigned'} {opts.output}")
 
 
+def pe_read_single_section(section, opts, name, text, obj):
```

Comment:
What's the reason to pass in the return type here as `obj`? If there's no deeper reason and with `pe_read_sections` seeming to be the only user, just return a dict here and update `json_obj` in `pe_read_sections` with it.